options/options.go

```diff
@@ -41,8 +42,8 @@ type Options struct {
 	ResultsFile string
 	ChecksToRun []string
 	Metadata    []string
+	Depth       int
```

Comment:
just being pedantic: why not call this `number-of-commits`. depth probably work, idk. @olivekl could you advise?

Reply:
+1, `depth` was not self-explanatory to me, `NumberOfCommits` is.

Reply:
sure thing, so having it like this `--number-of-commits int   number of commits to check, commits begin backwards from the HEAD` on the command line? and naming the variable in the code NumerOfCommits

Reply:
LGTM.
/cc @azeemsgoogle @naveensrinivasan @spencerschrock @raghavkaul no objections?